packages/terra-list/src/ListItem.scss

```diff
@@ -1,30 +1,23 @@
 @import '~terra-mixins';
 @import './variables';
 
-.terra-ListItem-hasChevron {
-  display: block;
-  position: relative; // relative position so the chevron can use absolute position to vertically center itself
+.terra-ListItem {
+  display: table;
+  outline: none;
+  width: 100%;
+}
 
-  [dir='ltr'] & {
-    @include terra-padding-end(20px);
-  }
+.terra-ListItem > * {
+  display: table-cell;
```

Comment:
When we have a chevron, we could use the arrange component here or create explicit classes / DOM for content and icon holders rather than adding the `*` selector. 
The selector above ends up styling more than it should setting `display: table-cell` to any children within the ListItem modifying the display of components in ways we might not expect.